src/components/WidgetContainer/widget-container.jsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import { closeIcon, profile } from "../../assests/index";
+import WidgetButton from "../WidgetButton/widget-button";
```

Comment:
Once you make the change in widget-button.jsx, this will need to become:

```suggestion
import { WidgetButton } from "../WidgetButton/widget-button";
```